======================================== 
Start time: 2025/04/28 周一 23:33:12.47 
Using conda environment: pytorch_gpu 
Starting Python script execution... 
Traceback (most recent call last):
  File "C:\Users\ZY\Desktop\Forecaster\scripts\task_evaluate.py", line 24, in <module>
    Evaluate_Task.execute_eva() # 依据task_id自动选择类方法创建数据文件
  File "C:\Users\ZY\Desktop\Forecaster\scripts\..\src\model_evaluator\evaluate_src.py", line 88, in execute_eva
    return task_mapping[task_id]()
  File "C:\Users\ZY\Desktop\Forecaster\scripts\..\src\model_evaluator\evaluate_src.py", line 63, in eva_task1
    model_dic[station_id].load_state_dict(pth_dic[station_id])
  File "C:\ProgramData\Anaconda3\envs\pytorch_gpu\lib\site-packages\torch\nn\modules\module.py", line 2189, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for GRU_Encoder:
	Missing key(s) in state_dict: "gru.weight_ih_l0", "gru.weight_hh_l0", "gru.bias_ih_l0", "gru.bias_hh_l0", "fc_out.weight", "fc_out.bias". 
	Unexpected key(s) in state_dict: "model". 
 
End time: 2025/04/28 周一 23:33:16.42 
Script execution completed 
